### PR TITLE
Cipher

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

File: common/src/ciphers.rs [hunk 1]
```diff
@@ -0,0 +1,58 @@
+pub fn list_ciphers() -> String {
+    "caeser, shift".to_string()
+}
+
+pub struct Encryption {}
+
+pub trait Encrypt {
+    fn caeser(&self, msg: String) -> String;
```
Comment (adamhutchings) [inline common/src/ciphers.rs:8]: Caesar

File: common/src/ciphers.rs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    fn shift(&self, k: u32, msg: String) -> String;
+}
+
+impl Encrypt for Encryption {
+    fn caeser(&self, msg: String) -> String {
```
Comment (adamhutchings) [inline common/src/ciphers.rs:13]: Same thing

**Review by adamhutchings — CHANGES_REQUESTED** [review]
Typos

**Review by adamhutchings — APPROVED** [review]

File: common/src/ciphers.rs [hunk 3]
```diff
@@ -0,0 +1,58 @@
+pub fn list_ciphers() -> String {
+    "caesar, shift".to_string()
```
Comment (Camerooooon) [inline common/src/ciphers.rs:2]: AKA ROT-3 Right? (Just curious)

**Review by Camerooooon — COMMENTED** [review]
Looks good to me!

**Pull request merged**